composer.json

```diff
@@ -16,8 +16,13 @@
     ],
     "require": {
         "php": "^8.0.2",
+        "ext-ctype": "*",
+        "ext-filter": "*",
```

Comment:
Required in:

- `illuminate/routing`
- `illuminate/validation`
- `illuminate/support`
- `illuminate/http`
- `illuminate/pagination`

Optional in:

- `illuminate/queue`
- `illuminate/cache`
- `illuminate/database`